people/feeback.md

```diff
@@ -0,0 +1,64 @@
+# Feedback
+
+> [Giving feedback turns out to be the unnatural atomic building block atop which the unnatural skill set of management gets built](https://a16z.com/2012/10/17/making-yourself-a-ceo/)
```

Comment:
https://en.wikipedia.org/wiki/Strange_loop

Reply:
[One of the best books have ever read](https://en.wikipedia.org/wiki/I_Am_a_Strange_Loop).